apps/designer/app/designer/features/style-panel/controls/select/select-control.tsx

```diff
@@ -1,8 +1,19 @@
+import { cssVars } from "@webstudio-is/css-vars";
 import { Grid, Select } from "@webstudio-is/design-system";
 import { getFinalValue } from "../../shared/get-final-value";
 import { PropertyName } from "../../shared/property-name";
 import type { ControlProps } from "../../style-sections";
 
+const gridTemplateColumnsVar = cssVars.define("grid-template-columns");
+
+export const selectControlCssVars = ({
```

Comment:
This represents a higher level API for CSS vars for SelectControl. Think of it as of react's context-like way to provide styles to a child component that is nested in the tree and we have no direct access to it over props.